OpenWebUI update fails due to NodeJS heap limit

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

OpenWebUI

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

8.4.14

### 📝 Provide a clear and concise description of the issue.

Trying to run "update" returns an error about the heap limit. Increasing the number at https://github.com/community-scripts/ProxmoxVE/blob/f6ca81833e243434b0013385ebda435d152ba3ce/ct/openwebui.sh#L67 helps. Tried it by quickly changing it manually on my machine. You can probably figure out the right number better than I can.

### 🔄 Steps to reproduce the issue.

Try to run the "update" command.

### ❌ Paste the full error output (if available).

```console
<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----

 1: 0xe13fde node::OOMErrorHandler(char const*, v8::OOMDetails const&) [node]
 2: 0x11d5070 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node]
 3: 0x11d5347 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [node]
 4: 0x1402c05  [node]
 5: 0x1402c33  [node]
 6: 0x141bd0a  [node]
 7: 0x141eed8  [node]
 8: 0x1c84ba1  [node]
Aborted

[ERROR] in line 68: exit code 0: while executing command $STD npm run build
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/openwebui.sh
+++ b/ct/openwebui.sh
@@ -64,7 +64,7 @@ function update_script() {
   fi
   systemctl stop open-webui.service
   $STD npm install --force
-  export NODE_OPTIONS="--max-old-space-size=3584"
+  export NODE_OPTIONS="--max-old-space-size=6000"
   $STD npm run build
   cd ./backend
   $STD pip install -r requirements.txt -U

--- a/install/openwebui-install.sh
+++ b/install/openwebui-install.sh
@@ -41,7 +41,7 @@ ENABLE_OLLAMA_API=false
 OLLAMA_BASE_URL=http://0.0.0.0:11434
 EOF
 $STD npm install --force
-export NODE_OPTIONS="--max-old-space-size=3584"
+export NODE_OPTIONS="--max-old-space-size=6000"
 $STD npm run build
 msg_ok "Installed Open WebUI"
 

